### ISSUE TITLE
[Feature Request]: Make JASP compatible with screen reader software

### ISSUE THREAD

**Issue by SwanseaUniversityTranscriptionCentre** [jasp-stats/jasp-issues#2196]
### Description

Develop JASP to make it compatible with screen reader software so that visually impaired users can use the software with their screen reader software. This is especially problematic for VI students using the software as part of their studies.

### Purpose

Accessibility, inclusivity, equality of access

### Is your feature request related to a problem?

JASP currently isn't compatible with screen reader software meaning visually impaired users and others who use screen readers for other disability related reasons cannot use JASP with their screen reader software

### Is your feature request related to a JASP module?

Unrelated

### Describe the solution you would like

Compatibility with screen reader software

### Describe alternatives that you have considered

Currently considering options such as a reader/lab assistant to work with the student but this doesn't allow the student to access JASP independently

**Comment by tomtomme** [comment]
closing, since this is a duplicate of https://github.com/jasp-stats/jasp-issues/issues/1783
